selenium-system-test/src/test/scala/hts/utils/Helpers.scala

```diff
@@ -17,9 +17,22 @@
 package hts.utils
 
 import hts.pages.Page
+import org.openqa.selenium.support.ui.{ExpectedConditions, WebDriverWait}
 import org.openqa.selenium.WebDriver
 
 object Helpers extends Page {
 
-  def isTextOnPage(text: String)(implicit driver: WebDriver): Boolean = text.r.findAllIn(getPageContent).nonEmpty
+  def isTextOnPage(text: String)(implicit driver: WebDriver): Boolean = {
+    val textPresent = text.r.findAllIn(getPageContent).nonEmpty
+    if (!(textPresent)) {
+      println("Text not found: " + text)
+    }
+    textPresent
+  }
+
+  def isActualUrlExpectedUrl(expectedUrl: String)(implicit driver: WebDriver): Boolean = {
```

Comment:
this method never actually returns `false`. What you could probably do to make this a bit cleaner is to do something like this:
```scala
def checkIsActualUrlExpectedUrl(expectedUrl: String)(implicit driver: WebDriver): Unit = {
 val wait = new WebDriverWait(driver, 20)
  wait.until(ExpectedConditions.urlContains(expectedUrl))
 }
```